Makefile

```diff
@@ -32,16 +32,17 @@ UNIT_TEST_PACKAGES ?= $$($(GO) list ./... | grep -vE "$(UNIT_TEST_PACKAGE_EXCLUS
 BRANCH_NAME?=main
 ifneq ($(PULL_BASE_REF),) # PULL_BASE_REF originates from prow
 	BRANCH_NAME=$(PULL_BASE_REF)
+	DEV_GIT_VERSION ?= $(shell source ./scripts/eksa_version.sh && eksa-version::get_next_eksa_version_for_ancestor $(PULL_BASE_REF))-dev+latest
```

Comment:
It may be helpful if you left a comment of an example value for this variable.

Reply:
added

Reply:
Thanks!